lib/email/reader.rb

```diff
@@ -0,0 +1,53 @@
+module Email
+  class Reader
+    EMAIL_REGEX = /(.*)--inline.html/
+    PATH_TO_EMAILS = '/emails'.freeze
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.